<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimaPag</class>
 <widget class="QMainWindow" name="PrimaPag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>551</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PrimaPag</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="p1_logare">
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="parola">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="l_username">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_intro">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>471</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Bun venit in aplicatia Apeluri de urgenta!</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_parola">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Parola</string>
      </property>
     </widget>
     <widget class="QPushButton" name="autentificare">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>310</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Autentificare</string>
      </property>
     </widget>
     <widget class="QLabel" name="l">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>390</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nu ai cont? </string>
      </property>
     </widget>
     <widget class="QPushButton" name="inregistrare">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>390</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <italic>true</italic>
        <underline>true</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Inregistraza-te!</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sch_parola">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>420</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <italic>true</italic>
        <underline>true</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Mi-am uitat parola!</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="p_mesaj_cerere">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cererea dumneavoastra a fost trimisa!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="spre_autentificare">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>430</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Inapoi la autentificare</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="p_sch_parola">
     <widget class="QLineEdit" name="email_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>199</y>
        <width>63</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Email-ul:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>291</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Vom trimite in scurt timp un email!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sch_parola_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>360</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Trimite cerere</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="parola_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>310</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>285</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Noua parola:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="p3_inregistrare">
     <widget class="QLineEdit" name="username_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="parola_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="nume">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenume">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="email">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>270</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="l_user">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_parola_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Parola:</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_nume">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nume:</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_prenume">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenume:</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_email">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_telefon">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>320</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Telefon:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="telefon">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="inregistrare_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>370</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Inregistrare</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="p2_client">
     <widget class="QLabel" name="us_pag_client">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>User</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>711</width>
        <height>491</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pag_client">
       <attribute name="title">
        <string>Apelare</string>
       </attribute>
       <widget class="QPushButton" name="apel_112">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>110</y>
          <width>91</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>112</string>
        </property>
       </widget>
       <widget class="QPushButton" name="apel_politie">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>250</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Politie</string>
        </property>
       </widget>
       <widget class="QPushButton" name="apel_pompieri">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>250</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pompieri</string>
        </property>
       </widget>
       <widget class="QPushButton" name="apel_salvamont">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>250</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Salvamont</string>
        </property>
       </widget>
       <widget class="QPushButton" name="apel_ambulanta">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>360</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ambulanta</string>
        </property>
       </widget>
       <widget class="QPushButton" name="apel_jandarmerie">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>360</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Jandarmerie</string>
        </property>
       </widget>
       <widget class="QPushButton" name="apel_smurd">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>360</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Smurd</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>220</y>
          <width>601</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Stii de ce ai nevoie? Alege tipul echipajului pentru a fi trimis direct la dispecerul specializat!</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="istoric_ap_client">
       <attribute name="title">
        <string>Istoric apeluri</string>
       </attribute>
       <widget class="QTableWidget" name="tbl_istoric_client">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>691</width>
          <height>451</height>
         </rect>
        </property>
        <property name="rowCount">
         <number>99</number>
        </property>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>Data si ora</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tipul apelului</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="p_dispecer">
     <widget class="QTabWidget" name="tab_dispecer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>501</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="detalii">
       <attribute name="title">
        <string>In asteptare</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="istoric_dispecer">
       <attribute name="title">
        <string>Istoric apeluri</string>
       </attribute>
       <widget class="QTableWidget" name="tbl_istoric_dispecer">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>1</y>
          <width>681</width>
          <height>461</height>
         </rect>
        </property>
        <property name="rowCount">
         <number>100</number>
        </property>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>Nume</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Prenume</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>username</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Data si ora</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Detalii adaugate</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fisierul audio</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="utilizatori">
       <attribute name="title">
        <string>Utilizatori</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="p_administrator">
     <widget class="QTabWidget" name="tab_admin">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>501</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="istoric_administrator">
       <attribute name="title">
        <string>Istoric apeluri</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="utilizatori_2">
       <attribute name="title">
        <string>Utilizatori</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="dispeceri">
       <attribute name="title">
        <string>Dispeceri</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="cereri_inregistrari">
       <attribute name="title">
        <string>Inregistrari</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="cereri_stergeri">
       <attribute name="title">
        <string>Stergeri</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
